client/blocks/product-selector/index.jsx

```diff
@@ -579,36 +603,34 @@ export class ProductSelector extends Component {
 					purchase={ purchase }
 					subtitle={ this.getSubtitleByProduct( product ) }
 				>
-					{ hasProductPurchase && this.renderManageButton( product, purchase ) }
-					{ ! hasProductPurchase && ! isCurrent && (
+					{ selectedSiteSlug &&
+						hasProductPurchase &&
+						isCurrent &&
+						this.renderManageButton( product, purchase ) }
```

Comment:
Unfortunately, this seems to revert #41262 at `/plans/monthly/<site>`:

<img width="1074" alt="Screen Shot 2020-04-27 at 3 59 41 PM" src="https://user-images.githubusercontent.com/4044428/80425040-224f7900-88a0-11ea-8900-5315b6f810ec.png">

Reply:
True, good catch!  `/monthly` is broken for yearly purchases and vice versa. Fixed!